nautobot/dcim/templates/dcim/powerpanel_retrieve.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'generic/object_retrieve.html' %}
+{% load helpers %}
+{% block extra_breadcrumbs %}
+    <li>{{ object.location|hyperlinked_object }}</li>
```

Comment:
We should fix the issue that `NautobotUIViewSet` isn't correctly detecting and rendering `powerpanel.html`, rather than introducing a new, redundant template as a part of this fix. See discussion on our Slack from Monday.

Reply:
Yeah, we need to refactor the logic in `get_base_template()` to detect both `object.html` and `object_retrieve.html`. Let me take a stab at this.

Reply:
Was speaking to Hanlin about this, imho we don't make it work for both, if I am looking for the template, to me it would be better to have the single standard vs any actual or perceived magic that future folks need to find.

Reply:
The alternative is this mess of duplicate/redundant files though, where we have both `foo.html` (for backward compatibility with apps) and `foo_retrieve.html` (for any custom templating logic) and it's even more confusing to anyone looking at the code base.

Reply:
These are only until 3.0. I don't think we should have reasonable expectations that these templates should be considered in the [public interface](https://docs.nautobot.com/projects/core/en/stable/development/core/#public-interface).

Reply:
The problem is that IIRC if there are Apps that are adding custom tabs to an existing object detail view, in at least *some* cases (thinking of Device in particular here) there has been a requirement for the custom tab's template to inherit from the base view template.

Personally, I consider the introduction of the secondary `foo_retrieve.html` pattern in NautobotUIViewSet to have been a misstep on our part and would prefer not to propagate it as a requirement.